clients/clients-integration-tests/src/test/java/org/apache/kafka/clients/MetricsDuringTopicCreationDeletionTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients;
+
+import org.apache.kafka.common.test.ClusterInstance;
+import org.apache.kafka.common.test.api.ClusterConfigProperty;
+import org.apache.kafka.common.test.api.ClusterTest;
+import org.apache.kafka.common.test.api.Type;
+import org.apache.kafka.server.config.ReplicationConfigs;
+import org.apache.kafka.server.config.ServerConfigs;
+import org.apache.kafka.server.config.ServerLogConfigs;
+import org.apache.kafka.server.metrics.KafkaYammerMetrics;
+
+import com.yammer.metrics.core.Gauge;
+
+import java.io.Closeable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MetricsDuringTopicCreationDeletionTest {
+
+    private static final String TOPIC_NAME_PREFIX = "topic";
+    private static final int TOPIC_NUM = 2;
+    private static final int CREATE_DELETE_ITERATIONS = 3;
+    private static final short REPLICATION_FACTOR = 1;
+    private static final int PARTITION_NUM = 3;
+
+    private final ClusterInstance clusterInstance;
+    private final List<String> topics;
+    private volatile boolean running = true;
+
+    public MetricsDuringTopicCreationDeletionTest(ClusterInstance clusterInstance) {
+        this.clusterInstance = clusterInstance;
+        this.topics = new ArrayList<>();
+        for (int n = 0; n < TOPIC_NUM; n++) {
+            topics.add(TOPIC_NAME_PREFIX + n);
+        }
+    }
+
+    /*
+     * Checking all metrics we care in a single test is faster though it would be more elegant to have 3 @Test methods
+     */
+    @ClusterTest(
+        types = {Type.KRAFT},
+        serverProperties = {
```

Comment:
Could you please configure `brokers = 1` to ensure it does not impacted by https://issues.apache.org/jira/browse/KAFKA-4605?